jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/dao/RDSDBManager.java

```diff
@@ -207,4 +213,43 @@ public static void insertRecord(String tableName, Map<String, String> data) {
             LOGGER.error("Error Executing Query", e);
         }
     }
+
+    public static boolean isFirstTimePluginExecuted(String dataSource, List<PolicyTable> policyList) {
+        String recordsCountQuery = "SELECT COUNT(*) AS count FROM cf_PolicyTable WHERE assetGroup = ?";
```

Comment:
To check if plugin is created, I think we can check if there is an entry in cf_Accounts table. But I 'll let you decided which approach is better.

Reply:
Yes that is another way of checking plugin created already. But here we are mostly dealing with Policy case , it would make more sense to use cf_policy table.